app/views/transactions/create.js.erb

```diff
@@ -1,15 +1,20 @@
 <% if @transaction.persisted? %>
   if ($('.transactions').size() == 0) {
-    // need to reload because there is no ransack initialized & no transactions table
-    location.reload();
+    $('.transactions_list').
+      html("<%=j render partial: 'transactions/transactions_table',
+        locals: { transactions:
+          Kaminari.paginate_array([@transaction]).page(params[:page]).per(50),
```

Comment:
@Gataniel why it's here? I don't understand why do you need to render single transaction as table. let's discuss it. It looks wrong.